src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -26,6 +27,8 @@ public SimulationCache(WorldGenerationSettings worldSettings)
             this.worldSettings = worldSettings;
         }
 
+        public bool DayNightCycleEnabled => worldSettings.DayNightCycleEnabled;
```

Comment:
Why is this property here? This is not even cached here so this looks pretty dirty to me in terms of logically accessing the world settings through a cache that doesn't even cache the value. What I mean is that this feels like the wrong place to put this. Why aren't things just directly using the world settings and getting the bool from there?

Reply:
Are you suggesting making worldSettings public in this class or passing worldSettings to the other classes here?

Reply:
World settings should be passed around where needed. It's not the cache's job to allow other parts of the code get the settings through the cache object.